Docs/Book/Cookbook.md

```diff
@@ -409,6 +409,8 @@ The code:
     fp = numpy.zeros((1,))
     DataStructs.ConvertToNumpyArray(AllChem.GetMorganFingerprintAsBitVect(m5, 2), fp)
 
+    # Reshape to a matrix for one sample
```

Comment:
I've been dealing with these warnings by just doing:
`rf.predict((fp,))`
Doesn't that seem clearer?